viz.yaml

```diff
@@ -1,333 +1,351 @@
 cvizlab: "0.2.0"
 info:
-  id: how-gdp
-  name: How GDP Works 
-  date: 2017-09-19
-  publish-date: 2018-10-12
-  path: /how-gdp
-  analytics-id: 
-  description: >-
-    Explanation of how the Geo Data Portal works.
-  keywords: >-
-    USGS, VIZLAB, GDP
-  audience: General public
+id: how-gdp
+name: How GDP Works 
```

Comment:
If this is the actual title, we probably shouldn't use an acronym

Reply:
This is what ends up in the tab on Google chrome. Is that the "actual title" ?